### ISSUE TITLE
WCAG 1.1.1 Non-text Content

### ISSUE THREAD

**Issue by ToneMereteBruvik** [ufal/dspace-angular#33]
These icons missing alternative text (reported by Lighthouse): 

![Image](https://github.com/user-attachments/assets/2bbda46f-656c-4e34-922d-ea864ba792f4)
